CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## 3.5.1
```

Comment:
do we need a new release?

Reply:
Actually we don't, it can perfectly wait for the next one. I included it because I was releasing before. Thanks!

Reply:
Reverted here: [4f330d0](https://github.com/newrelic/nri-kubernetes/pull/570/commits/4f330d0714f2de0937805afb3127099559bc2a3e)